src/schema/artwork/index.js

```diff
@@ -152,13 +152,16 @@ export const artworkFields = () => {
     contact_message: {
       type: GraphQLString,
       description: "Pre-filled inquiry text",
-      resolve: ({ partner, availability }) => {
+      resolve: ({ partner, availability_hidden, availability }) => {
         if (partner && partner.type === "Auction") {
           return [
             "Hello, I am interested in placing a bid on this work.",
             "Please send me more information.",
           ].join(" ")
         }
+        if (availability_hidden) {
```

Comment:
This seems sort of non-sensical, but maybe this conditional should go first? In this case, a work that is at an auction (but marked to be 'hidden'), will still include that auction-specific text.

So, just to be safe/thorough, maybe we move this up before that?

Reply:
I actually thought about that and kinda decided that auction takes precedence. Like if the artwork is from auction house - that means it should definitely be biddable one way or the other.... Though maybe that is not the case. I'm happy to move the check up if you still think it's more logical-:)

Or I guess we could ask somebody from #product what should happen when artwork is from Auction partner and still has hidden availability (just typing it - it should be very unlikely case:))

Reply:
> Or I guess we could ask somebody from #product what should happen when artwork is from Auction partner and still has hidden availability (just typing it - it should be very unlikely case:))

What was the outcome of this?

Reply:
Ok, @mzikherman. @alloy - the answer to above question is that the flow that allows inquiries on works from Auction partners is completely deprecated in current UI so it's truly does not matter what takes precedence here.

Ideally we should remove the code that takes this flow into account across the board, but that should probably be a separate ticket either way.

This PR should be good to merge as is.